pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
-            "format": "date-time"
+            "example": "ok",
+            "enum": [
+              "ok",
+              "not ready",
+              "unhealthy",
+              "maintenance"
+            ],
+            "description": "The status of the node."
           },
-          "received_at": {
+          "uptime": {
             "type": "string",
-            "format": "date-time"
+            "format": "duration",
+            "example": "25h33m29.4860s",
+            "description": "The duration since the node was last started."
+          },
+          "version": {
+            "type": "string",
+            "format": "semver",
+            "example": "1.0.0",
+            "description": "The version of the node."
           }
         }
       },
-      "Pending": {
+      "APIAuthentication": {
         "type": "object",
+        "description": "The request object for the authentication endpoint.",
         "properties": {
-          "envelope_id": {
-            "type": "string"
-          },
-          "received_by": {
-            "type": "string"
+          "client_id": {
+            "type": "string",
+            "description": "The client ID for authentication."
           },
-          "received_at": {
+          "client_secret": {
+            "type": "string",
+            "description": "The client secret for authentication."
+          }
+        }
+      },
+      "LoginReply": {
+        "type": "object",
+        "description": "The response object for the authentication endpoint.",
+        "properties": {
+          "access_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The access token for the authenticated session."
           },
-          "message": {
-            "type": "string"
+          "token_type": {
+            "type": "string",
+            "description": "The type of the access token."
           },
-          "reply_not_after": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for obtaining a new access token."
           },
-          "reply_not_before": {
+          "expires_in": {
+            "type": "integer",
+            "description": "The duration in seconds until the access token expires."
+          }
+        }
+      },
+      "ReauthenticateRequest": {
+        "type": "object",
+        "description": "The request object for the reauthentication endpoint.",
+        "properties": {
+          "refresh_token": {
             "type": "string",
-            "format": "date-time"
+            "description": "The refresh token for reauthentication."
+          }
+        }
+      },
+      "PageQuery": {
+        "type": "object",
+        "description": "The request object for paginated endpoints.",
+        "properties": {
+          "page_size": {
+            "type": "integer",
+            "format": "int32",
```

Comment:
The `page_size` has an `int` type [here](https://github.com/trisacrypto/envoy/blob/43266936ce5b681ee8ec042510072195d269823d/pkg/web/api/v1/api.go#L94). I believe the format should be removed or updated since `int32` isn't specified.